workflows/prognostic_c48_run/runtime/steppers/prescriber.py

```diff
@@ -0,0 +1,196 @@
+from typing import Sequence, MutableMapping, Mapping, Hashable, Tuple, Optional
+import dataclasses
+from datetime import timedelta
+import logging
+import intake
+import cftime
+import xarray as xr
+from runtime.types import State, Diagnostics, Tendencies
+import fv3gfs.util
+from vcm.catalog import catalog as CATALOG
+from vcm.safe import get_variables
+
+
+logger = logging.getLogger(__name__)
+
+QuantityState = MutableMapping[Hashable, fv3gfs.util.Quantity]
+
+
+def get_timesteps(
+    init_time: cftime.DatetimeJulian, timestep_seconds: float, n_timesteps: int
+) -> Sequence[cftime.DatetimeJulian]:
+    """Get sequence of model timesteps
+    
+    Args
+        init_time (cftime.DatetimeJulian): model run start time
+        timestep_seconds (float): model timestep
+        n_timesteps: number of timesteps in model run
+    
+    Returns: Sequence of cftime.DatetimeJulian objects corresponding to ends
+        of model run timesteps
+    
+    """
+    return [
+        init_time + timedelta(seconds=timestep_seconds * i)
+        for i in range(1, n_timesteps + 1)
+    ]
+
+
+@dataclasses.dataclass
+class PrescriberConfig:
+    """Configuration for a prescriber object to set states in the model from an external source
+
+    Attributes:
+        dataset_key (str): location of the dataset that provides prescribe values;
+            the routine first will try `dataset_key` as a `vcm.catalog` key, and if
+            not present it will attempt to use it as a (local or remote) path to
+            a zarr dataset
+        variables (Sequence[str]): sequence of "standardized" ("_coarse" suffix removed)
+            variable names to prescribe
+        rename (Mapping[Hashable, Hashable]): mapping of "standardized" ("_coarse"
+            suffix removed) names in the external dataset to variable names desired
+            for the runfile
```

Comment:
could you remove references to the `_coarse` suffix which no longer seems relevant. Also in principle we don't really need the renaming capability anymore but I'm okay to leave it in. I'm not really sure what `"standardized"` means here.